### PR TITLE
update the pr-agent name to title case

### PR DESCRIPTION
**Pull request by tombrewsviews**
A question related to the naming convention consistency - do we want to keep the dash/hyphen in the name?

Right now, I updated to PR-Agent, but we list tools like PR Review and PR Q&A without the dashes. @almog-lv @GadiZimerman @coditamar

### CODE REVIEW

**Comment by okotek** [comment]
PR-Agent without a dash doesn't work well, too similar to "pregnant". 
Codium-ai is the org name in GitHub, it's consistent with other orgs styling.
pr-agent in small letters is the project name. It's common for these project to be in small letters.

@tombrewsviews

**Comment by mrT23** [comment]
i also think it should be in capitals
"PR-Agent"

pr-agent is just wierd. 99.9% of opensource projects are not named with lowercase

**Comment by okotek** [comment]
https://github.com/tensorflow/tensorflow
https://github.com/pytorch/pytorch
https://github.com/mongodb/mongo
https://github.com/mysql/mysql-server
https://github.com/facebook/react

While I don't mind that the styling in the docs will be capitalized, I was referring to the GitHub slug which should be lower case (as it is today)

File: README.md [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
 </div>
 
-[![GitHub license](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/Codium-ai/pr-agent/blob/main/LICENSE)
+[![GitHub license](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/Codium-ai/PR-Agent/blob/main/LICENSE)
```
Comment (okotek) [inline README.md:10]:
```suggestion
[![GitHub license](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/Codium-ai/pr-agent/blob/main/LICENSE)
```

File: README.md [hunk 2]
```diff
@@ -50,13 +50,13 @@ To request a review for a PR, or ask a question about a PR, you can run directly
 1. To request a review for a PR, run the following command:
 
 ```
-docker run --rm -it -e OPENAI.KEY=<your key> -e GITHUB.USER_TOKEN=<your token> codiumai/pr-agent --pr_url <pr url>
+docker run --rm -it -e OPENAI.KEY=<your key> -e GITHUB.USER_TOKEN=<your token> codiumai/PR-Agent --pr_url <pr url>
```
Comment (okotek) [inline README.md:53]:
```suggestion
docker run --rm -it -e OPENAI.KEY=<your key> -e GITHUB.USER_TOKEN=<your token> codiumai/pr-agent --pr_url <pr url>
```

File: README.md [hunk 3]
```diff
@@ -50,13 +50,13 @@ To request a review for a PR, or ask a question about a PR, you can run directly
...
 ```
 
 2. To ask a question about a PR, run the following command:
 
 ```
-docker run --rm -it -e OPENAI.KEY=<your key> -e GITHUB.USER_TOKEN=<your token> codiumai/pr-agent --pr_url <pr url> --question "<your question>"
+docker run --rm -it -e OPENAI.KEY=<your key> -e GITHUB.USER_TOKEN=<your token> codiumai/PR-Agent --pr_url <pr url> --question "<your question>"
```
Comment (okotek) [inline README.md:59]:
```suggestion
docker run --rm -it -e OPENAI.KEY=<your key> -e GITHUB.USER_TOKEN=<your token> codiumai/pr-agent --pr_url <pr url> --question "<your question>"
```

File: README.md [hunk 4]
```diff
@@ -74,7 +74,7 @@ Possible questions you can ask include:
 1. Clone this repository:
 
 ```
-git clone https://github.com/Codium-ai/pr-agent.git
+git clone https://github.com/codium-ai/pr-agent.git
```
Comment (okotek) [inline README.md:77]:
```suggestion
git clone https://github.com/Codium-ai/pr-agent.git
```

File: README.md [hunk 5]
```diff
@@ -136,7 +136,7 @@ WEBHOOK_SECRET=$(python -c "import secrets; print(secrets.token_hex(10))")
 4. Clone this repository:
 
 ```
-git clone https://github.com/Codium-ai/pr-agent.git
+git clone https://github.com/codium-ai/pr-agent.git
```
Comment (okotek) [inline README.md:139]:
```suggestion
git clone https://github.com/Codium-ai/pr-agent.git
```

File: README.md [hunk 6]
```diff
@@ -154,8 +154,8 @@ cp pr_agent/settings/.secrets_template.toml pr_agent/settings/.secrets
 6. Build a Docker image for the app and optionally push it to a Docker repository. We'll use Dockerhub as an example:
 
 ```
-docker build . -t codiumai/pr-agent:github_app --target github_app -f docker/Dockerfile
-docker push codiumai/pr-agent:github_app  # Push to your Docker repository
+docker build . -t codiumai/PR-Agent:github_app --target github_app -f docker/Dockerfile
```
Comment (okotek) [inline README.md:157]:
```suggestion
docker build . -t codiumai/pr-agent:github_app --target github_app -f docker/Dockerfile
```

File: README.md [hunk 7]
```diff
@@ -154,8 +154,8 @@ cp pr_agent/settings/.secrets_template.toml pr_agent/settings/.secrets
...
+docker push codiumai/PR-Agent:github_app  # Push to your Docker repository
```
Comment (okotek) [inline README.md:158]:
```suggestion
docker push codiumai/pr-agent:github_app  # Push to your Docker repository
```

**Review by okotek — CHANGES_REQUESTED** [review]

**Comment by zmeir** [comment]
Preparing review...

File: README.md [hunk 8]
```diff
@@ -24,22 +24,22 @@ CodiumAI `pr-agent` is an open-source tool aiming to help developers review PRs
 
 ## Live demo
 
-Experience GPT-4 powered PR review on your public GitHub repository with our hosted pr-agent. To try it, just mention `@CodiumAI-Agent` in any PR comment! The agent will generate a PR review in response.
+Experience GPT-4 powered PR review on your public GitHub repository with our hosted PR-Agent. To try it, just mention `@CodiumAI-Agent` in any PR comment! The agent will generate a PR review in response.
 
-![Review generation process](./pics/pr-agent-review-process1.gif)
+![Review generation process](./pics/PR-Agent-review-process1.gif)
```
Comment (okotek) [inline README.md:29]:
```suggestion
![Review generation process](./pics/pr-agent-review-process1.gif)
```

**Review by okotek — CHANGES_REQUESTED** [review]

**Review by okotek — APPROVED** [review]

**Review by okotek — APPROVED** [review]

**Pull request merged**